build.gradle

```diff
@@ -811,26 +819,35 @@ task gatkWDLGenValidation(dependsOn: [gatkWDLGen, shadowJar]) {
         def wdlFiles = fileTree(dir: wdlGenFolder).filter {
             f -> !f.getAbsolutePath().endsWith(".html") && !f.getAbsolutePath().endsWith(".json")
         }
-        wdlFiles.any() { wdlFile ->
-            final womtoolLocation = System.getenv('WOMTOOL_JAR')
-            final validateWDLCommand = "java -jar $womtoolLocation validate $wdlFile"
-            execWDLValidation(validateWDLCommand)
-        }
+        final womtoolLocation = System.getenv('WOMTOOL_JAR')
+//        wdlFiles.any() { wdlFile ->
+//            final validateWDLCommand = "java -jar $womtoolLocation validate $wdlFile"
+//            execWDLValidation(validateWDLCommand)
+//        }
```

Comment:
Looks like you left the womtool validation step commented out -- was this intentional?

Reply:
Oh good catch- it was a temporary shortcut and not intentional. Fixed.